controller.go

```diff
@@ -372,6 +372,12 @@ func syncPV(pv *PV) {
 				}
 			} else {
 				// The PV must have been created with this ptr; leave it alone.
```

Comment:
Okay, I'm sold -- but I will probably make a PR adding a comment explaining what 'available' means in this context